Update Components list in Kyma docs (#5686)
kyma-project/kyma: docs/kyma/03-01-components.md

The tracing in Kyma uses the [Jaeger](https://github.com/jaegertracing) distributed tracing system. Use it to analyze performance by scrutinizing the path of the requests sent to and from your service. This information helps you optimize the latency and performance of your solution.

## API Gateway

> The API Gateway aims to provide a set of functionalities which allow developers to expose, secure, and manage their API's in an easy way. The main elemennt of the API Gateway is the API Controller which exposes services in Kyma.

Reviewer: ```suggestion
The API Gateway aims to provide a set of functionalities which allow developers to expose, secure, and manage their APIs in an easy way. The main element of the API Gateway is the API Controller which exposes services in Kyma.
```

## Logging

Logging in Kyma uses [Loki](https://github.com/grafana/loki), a Prometheus-like log management system.

## Backup

Kyma integrates with [Velero](https://github.com/heptio/velero/) to provide backup and restore capabilities for Kubernetes cluster resources. Once backed up, Velero stores the resources in buckets of [supported cloud providers](https://velero.io/docs/v1.0.0/support-matrix/).

## Console

> The Console is a web-based administrative UI for Kyma. It uses the [Luigi framework](https://github.com/SAP/luigi) to allow you to seamlessly extend the UI content with custom micro frontends. The Console has a dedicated backend service, Console Backend Service, which provides an API tailor-made to each view of the Console UI.

Reviewer: ```suggestion
The Console is a web-based administrative UI for Kyma. It uses the [Luigi framework](https://github.com/SAP/luigi) to allow you to seamlessly extend the UI content with custom micro frontends. The Console has a dedicated Console Backend Service which provides a tailor-made API for each view of the Console UI.
```

## Asset Store

The Asset Store is a flexible, scalable, multi-cloud, and location-independent Kubernetes-native solution for storing assets, such as documents, files, images, API specifications, and client-side applications. The Asset Store consists of Minio and custom resources. The custom resources are managed by a controller that communicates through Minio Gateway with external cloud providers.

## Headless CMS

> The Headless CMS is a Kubernetes native Content Management Systems (CMS) that provides a way of storing and managing raw content, and exposing it through an API. The Headless CMS is an abstraction on top of the Asset Store which provides a data layer. The Headless CMS has no dedicated presentation layer but it is integrated with the Console UI that consumes content stored using Headless CMS.

Reviewer: ```suggestion
The Headless CMS is a Kubernetes-native Content Management System (CMS) that provides a way of storing and managing raw content, and exposing it through an API. The Headless CMS is an abstraction on top of the Asset Store which provides a data layer. The Headless CMS has no dedicated presentation layer but it is integrated with the Console UI that consumes content stored using Headless CMS.
```

The tracing in Kyma uses the [Jaeger](https://github.com/jaegertracing) distributed tracing system. Use it to analyze performance by scrutinizing the path of the requests sent to and from your service. This information helps you optimize the latency and performance of your solution.

## API Gateway

The API Gateway aims to provide a set of functionalities which allow developers to expose, secure, and manage their API's in an easy way. The main elemennt of the API Gateway is the API Controller which exposes services in Kyma.

## Logging

Logging in Kyma uses [Loki](https://github.com/grafana/loki), a Prometheus-like log management system.

## Backup

Kyma integrates with [Velero](https://github.com/heptio/velero/) to provide backup and restore capabilities for Kubernetes cluster resources. Once backed up, Velero stores the resources in buckets of [supported cloud providers](https://velero.io/docs/v1.0.0/support-matrix/).

## Console

> The Console is a web-based administrative UI for Kyma. It uses the [Luigi framework](https://github.com/SAP/luigi) to allow you to seamlessly extend the UI content with custom micro frontends. A backend service which provides an API for all views of the Console UI is the Console Backend Service.

Reviewer 2: Console uses a dedicated backend service (Console Backend Service) which provides an API for all views of the Console UI.

## Asset Store

> The Asset Store is a flexible, scalable, multi-cloud, and location-independent Kubernetes-native solution for storing assets, such as documents, files, images, API specifications, and client-side applications. Its main components are Asset and Bucket custom resources (CRs), Asset and Bucket Controllers, and Minio Gateway as a default storage solution. The Asset Store also allows you to use optional services, such as Validation, Mutation, and Metadata Services.

Reviewer 3: `The Asset Store also allows you to use optional services, such as Validation, Mutation, and Metadata Services.` - to much details, I would remove it
`Asset and Bucket Controllers` - it is one controller that manages all resources

so AssetStore is Minio and custom resources on top of it managed by a controller that communicates through  Minio Gateway with external cloud providers

## Headless CMS

> The Headless CMS is a new kind of Content Management Systems (CMS) that provides a way of storing and managing raw content, and exposing it through an API. It only ensures a database backend without providing a display layer. In Kyma, the Headless CMS allows you to upload multiple and grouped data for a given documentation topic and store them as Asset CRs in Minio buckets.

Reviewer 3: `is a new kind of Content Management Systems (CMS)` -> `is a Kubernetes native Content Management Systems (CMS)`

please rephrase 
```
It only ensures a database backend without providing a display layer. In Kyma, the Headless CMS allows you to upload multiple and grouped data for a given documentation topic and store them as Asset CRs in Minio buckets.
```

just write that hcms is an abstraction on top of AssetStore and this way, out of the box provides only a data layer. There is no separate dedicated presentation layer but it is integrated with Console UI that consumes content stored with hcms

## Headless CMS

> The Headless CMS is a Kubernetes native Content Management Systems (CMS) that provides a way of storing and managing raw content, and exposing it through an API. The Headless CMS is an abstraction on top of the Asset Store which provides a data layer. The Headless CMS has no dedicated presentation layer but it is integrated with the Console UI that consumes content stored using Headless CMS.

Reviewer 4: 😢
